scripts/populate_institutions.py

```diff
@@ -243,6 +243,18 @@ def main(default_args=False):
                 'email_domains': [],
                 'delegation_protocol': 'saml-shib',
             },
+            {
+                '_id': 'cord',
+                'name': 'Concordia College',
+                'description': '<a href="https://www.concordiacollege.edu/">Concordia College</a> | <a href="https://www.concordiacollege.edu/academics/library/">Carl B. Ylvisaker Library</a> | <a href="https://cord.libguides.com/?b=s">Research Guides</a>',
+                'banner_name': 'cord-banner.png',
+                'logo_name': 'cord-shield.png',
+                'login_url': None,
+                'logout_url': SHIBBOLETH_SP_LOGOUT.format(encode_uri_component('https://osf.io/goodbye')),
```

Comment:
https://github.com/CenterForOpenScience/osf.io/blob/5afbe5a548819b024bf40a39194788def2e18ff9/scripts/populate_institutions.py#L26

This `SHIBBOLETH_SP_LOGOUT` needs to be updated as a future improvement.

* Does `/Shibboleth.sso/Logout` clear both Shibboleth and CAS cookies? If not, need a fix.
* CAS / `pac4j` institutions do not need the extra step for cleaning Shibboleth cookie. CAS logout is good enough.
* OSF only needs to store the OSF URL `https://osf.io/goodbye` or institution required logout URLs such as `https://shibboleth.nyu.edu/idp/profile/Logout`. It is CAS's job to build the actual one.